### PR TITLE
Adding IncludeDefaultCfg and making corrections

### PR DESCRIPTION
**Pull request by AndrewYangMSFT**
1. Adding IncludeDefaultCfg to the table of the reg values that OEMs can customize under USBFN. Also removing unnecessary spaces from the raw code of the table
2. Removing a few reg key paths which I don't think necessary to mention.
3. Correcting some statements such as how the hardware ID is determined.
4. Correcting some reg key paths' formatting.

### CODE REVIEW

**Comment by duncanmacmichael** [comment]
Adding @billlattimer as I've moved to another team. Thanks!

File: windows-driver-docs-pr/usbcon/usb-registry-settings-for-a-function-controller-driver.md [hunk 1]
```diff
@@ -51,48 +51,46 @@ This table describes the subkeys that OEMs can modify under this key. More infor
...
-| **iSerialNumber**        | REG\_DWORD | OEM   | If this value is set to 0, then the device does not have a serial number. If this value is non-zero or does not exist, then the serial number is generated uniquely per device.                                                                                                                                            |
-| **CurrentConfiguration** | REG\_SZ    | OEM   | This string must correspond to the name of a configuration subkey. This string determines which configuration to use to build a configuration descriptor for USB device enumeration.                                                                                                                                       |
+| Value | Type | Owner | Description|
+|--|--|--|--|
+| **IncludeDefaultCfg** | REG\_DWORD | OEM | Set to 0 when OEMs do not want to include the interfaces of the Default configuration such as IpOverUsb or MTP. When ommited, the default value is 1. |
```
Comment (AndrewYangMSFT) [inline windows-driver-docs-pr/usbcon/usb-registry-settings-for-a-function-controller-driver.md]: the statement for the omitted case might be incorrect. I will double-check and update the PR.

Comment (AndrewYangMSFT) [reply windows-driver-docs-pr/usbcon/usb-registry-settings-for-a-function-controller-driver.md]: updated in commit #2

**Comment by DOMARS** [comment]
@EliotSeattle - I think this PR is ready to go, so will merge later today.

**Pull request merged**